QML_HUD/vbar.cpp

```
@@ -0,0 +1,36 @@
+
+VerticalBar::VerticalBar(QQuickItem* parent)
+    : AbstractControl(parent)
+{
+}
+
+void VerticalBar::paint(QPainter* painter)
+{
+    // TODO: angle instead factor <0;45> : default = 15
+    static const QPointF backgroundPoints[4] = {
+        QPointF(m_Size / 4.0 - m_Size * 0.2, 0),
+        QPointF(3.0 * m_Size / 4.0 + m_Size * 0.2, 0),
```

cppcheck (2.10-2)

[error] There is an unknown macro here somewhere. Configuration is required. If Q_PROPERTY is a macro then please configure it.

(unknownMacro)